sitecrafting/groot
# Lando/Webpack improvements

* Delete `print.js` and friends after each webpack build
* Fix CSS source maps
* Upgrade Landofile to support rc.2+

To test:

```
cd /path/to/your/groot/dev/install
git fetch --all
git checkout webpack-improvements
lando destroy
lando start
lando webpack # or `lando webpack --watch`
```

Then make changes in the JS/LESS as you normally would, and verify that everything is working.
